module/actor/roll-dice.js

```diff
@@ -2,17 +2,28 @@
 /* global ChatMessage, Roll, game */
 
 // Function to roll dice
-export async function rollDice (numDice, actor, label = '', difficulty = 0, useHunger = true) {
+// numDice = Number of dice the function will roll
+// actor = Actor's data
+// label = Text that appears at the head of the ChatMessage
+// difficulty = The amount of successes required for a given roll
+// useHunger = Will roll hunger dice, if true
+// increaseHunger = Will increase the actor's hunger if no successes are rolled, if true
+// subtractWillpower = Subtracts a point of willpower, always, if true
+export async function rollDice (numDice, actor, label = '', difficulty = 0, useHunger = true, increaseHunger = false, subtractWillpower = false) {
+  // Define the actor's current hunger
   let hungerDice
   if (useHunger) {
     hungerDice = Math.min(actor.data.data.hunger.value, numDice)
   } else {
     hungerDice = 0
   }
+
+  // Roll defining and evaluating
   const dice = numDice - hungerDice
   const roll = new Roll(dice + 'dvcs>5 + ' + hungerDice + 'dhcs>5', actor.data.data)
-  await roll.evaluate()
+  roll.evaluate()

```

Comment:
Does this work?

Reply:
Yes it does! In v9 roll.evaluate() is becoming asynchronous and thus the await isn't necessary, is what I gathered.